web_notify/models/res_users.py

```diff
@@ -32,6 +32,11 @@ def _compute_channel_names(self):
     notify_info_channel_name = fields.Char(compute="_compute_channel_names")
     notify_default_channel_name = fields.Char(compute="_compute_channel_names")
 
+    allow_native_notifications = fields.Boolean(
```

Comment:
I'm not sure if we need this at all - either the user has allowed desktop notifications, then I think we should use them too, or they didn't, and then we don't need this flag